### DISCUSSION TITLE
Security Onion Not Show Red Flag Allerts

### DISCUSSION THREAD

**Discussion by sinungp** [Security-Onion-Solutions/securityonion#6024] [Unsupported Versions] [answered]
hello I'm a new security onion user

I have a problem, because the warning menu does not issue a red flag when I do nmap

is there any solution i can do?

Thank you

![image](https://user-images.githubusercontent.com/82489803/139105003-843855d0-d826-4366-8b97-b8d84405d30c.png)

**Comment by dougburks** [comment, marked as answer]
From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:

Please include the following details where you can:

- Security Onion version as seen in the lower left corner of SOC and in /etc/soversion. For example: 2.3.70
- Install source. Did you install from our Security Onion ISO image or did you perform a network installation?
- If network install, did you install from CentOS 7 or Ubuntu 18.04?
- Install type. ex. eval, standalone, forward node, search node, etc
- Does so-status show all services running?
- Do you get any failures when you run salt-call state.highstate from the node?
- Does the grid status show any failures?

**Reply by sinungp** [reply]
Thanks for the repply

i will describe it 

I use security onions
- version 2.3.80
- I installed from an iso image
- standalone installation type
- my so-status all is ok
![so-status](https://user-images.githubusercontent.com/82489803/139772184-7138be0d-25b9-4646-833e-39215fe6dcd5.png)
- this is the salt-call state.highstate
![salt-call](https://user-images.githubusercontent.com/82489803/139772552-22a99e19-125b-4e67-bedf-589ae47bd84a.png)
- I don't understand the meaning of "grid status"

Thanks

**Reply by dougburks** [reply]
> * I don't understand the meaning of "grid status"

https://docs.securityonion.net/en/2.3/grid.html

Additional questions:

Do you have 2 network interfaces, one for management and one for sniffing?

If so, is the sniffing interface connected to a tap or span port?

**Reply by sinungp** [reply]
aa i understand now

i have 2 interface

eth0 i using it to management
and eth1 use to sensor interface / sniffing 

i use a local computer without virtual machine

thanks for your respons

**Reply by dougburks** [reply]
Is the sniffing interface connected to a tap or span port?

**Reply by sinungp** [reply]
Hello

sorry but i don't understand what is tap or span port

what I do is connect the sniffing interface to the main switch of my network, and the switch I have is a non-manage switch

thank you

**Reply by dougburks** [reply]
If your sniffing interface is connected to a normal port on a normal switch, then it is not seeing the network traffic that you want to see. You'll need a switch that can provide a span port or a separate tap:
https://docs.securityonion.net/en/2.3/hardware.html#packets

**Reply by sinungp** [reply]
Hello

i have connected port sniffing into span port on switch manage

but the same thing still happens, i can't get red flag when i do nmap or brute force

is there anything else I can do to fix this?

thank you for the response

**Reply by tomjones1977** [reply]
Try running tcpdump on your SO machine, then run something like ping from a client device to an IP like 8.8.8.8 - see if you can see the traffic in tcpdump.  If you can then the span port and the sniffing interface are configured correctly and the problem is elsewhere.  If you cannot see it then the problem is still in the network setup or the sniffing interface configuration.

**Reply by sinungp** [reply]
Hello

I've solved this problem
I double checked the span port configuration on the switch I have

thanks for all the replies and answers
